docs/source/it/_config.py

```diff
@@ -0,0 +1,14 @@
+# docstyle-ignore
+INSTALL_CONTENT = """
+# Transformers installation
+! pip install transformers datasets
+# To install from source instead of the last release, comment the command above and uncomment the following one.
```

Comment:
Those two comments could be translated in Italian as well :-)

Reply:
Done, I hadn't noticed them, thank you 😊 🌈